src/factories/buildCommand.js

```diff
@@ -2,12 +2,14 @@
 
 const _ = require('lodash');
 
-module.exports = (executable, options, optionMap, target) => {
+module.exports = function(executable, options, optionMap) {
```

Comment:
Yep, the changes is because the arguments object isn't available via the arrow syntax (it's like a function but not really a function). Ultimately, we would remove the use of arguments object and use rest parameters instead (https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/rest_parameters) but rest parameters aren't available in Node yet.